modules/ROOT/pages/installation-prerequisites.adoc

```diff
@@ -6,34 +6,71 @@ endif::[]
 +
 If you don't see any roles in the list, refer to the Salesforce knowledge article https://help.salesforce.com/articleView?language=en_US&type=1&mode=1&id=000322715[Add roles to the role hierarchy, role=external, window=_blank] for instructions to create a role in your organization.
+. Select *Salesforce CRM Content User*.
+. *Save*.
+
+== API Community Manager Entitlement
+
+Installation requires Anypoint API Community Manager entitlement for your Anypoint organization.
 
-.. Select *Salesforce CRM Content User*.
-.. Click *Save*.
-. The Anypoint API Community Manager entitlement for your Anypoint organization.
-+
 To check the entitlement:
 
-.. Log in to your Anypoint Platform organization.
-.. If you are using the US cloud, navigate to `+https://anypoint.mulesoft.com/accounts/api/profile+`.
+. Log in to your Anypoint Platform organization.
+. If you are using the US cloud, navigate to `+https://anypoint.mulesoft.com/accounts/api/profile+`.
 +
 If you are using the EU cloud, navigate to `+https://eu1.anypoint.mulesoft.com/accounts/api/profile+`.
 +
 If you are using MuleSoft Government Cloud, navigate to `+https://gov.anypoint.mulesoft.com/accounts/api/profile+`.
-.. Verify that under the *Entitlements* property, the `apiCommunityManager` value is `true`.
-. Identity provider data.
+. Verify that under the *Entitlements* property, the `apiCommunityManager` value is `true`.
+
+== A Domain with Enabled Communities
+
+To set up and register a domain:
+
+. Log in to your API Community Manager https://login.salesforce.com/[Salesforce Organization, role=external, window=blank].
+. Go to *Salesforce Set-up* by selecting the gear icon and selecting *Set-up*.
+. Search for *My Domain* using the *Quick Find* box and then select *My Domain*.
+. Enter a domain name for your organization.
++
+The name can include up to 34 letters, numbers, and hyphens.
+. Select *Check Availability*.
++
+If the name is already taken, choose a different one.
+. Select *Register Domain*.
+
+You will receive a notification email when your domain is registered and your
+subdomain name is ready for testing. This process can take a few minutes.
+You can continue with the next steps while you wait for the email.
+
+NOTE: After you complete API Community Manager installation and configuration, you can configure a full custom domain
+name, also known as a vanity domain. When you're ready, refer to the Salesforce knowledge article
+https://help.salesforce.com/articleView?id=000336819&type=1&mode=1[Run your Salesforce Community under a custom domain, role=external, window=blank].
+
+Using the domain name you selected in the previous step, follow the steps in
+https://help.salesforce.com/articleView?id=networks_enable.htm&type=5[Enable Salesforce Communities, role=external, window=blank]
+to enable communities inside your Salesforce organization.
+
+Wait to receive the notification email that your domain is registered, and then deploy your domain:
```

Comment:
This is confusing.
We say that you can continue with the next steps...but there are no steps until Line 69, and those are for deploying the domain (where is the registration, subdomain name, and testing?)
After telling the audience to wait for the registration email, we diverge into a note that is about what is essentially a separate topic about vanity naming, and then a para about enabling communities.

Reply:
What we have here is ...:

. Select *Register Domain*.
Wait a few minutes
Continue with these steps while you wait:  (no steps)
NOTE: vanity name
LINK: enable communities
Wait for registration (repeat) 
Then deploy:  (separate task?)
. Log in
. Go to
.
.
.